crates/hir-ty/src/consteval/tests.rs

```diff
@@ -851,6 +851,7 @@ fn ifs() {
 fn loops() {
     check_number(
         r#"
+    //- minicore: add, builtin_impls
```

Comment:
Unlike our old implementation, the rustc implementation for operators require the presence of the trait and an impl even for builtin types. So we need to add them in a bunch of tests.